whimsical-woodpeckers/www/src/py/pyvue.py

```diff
@@ -0,0 +1,110 @@
+# __pragma__ ('js', "{}", "import * as VuePkg from 'vue';")
```

Comment:
the whole pyvue context should have been documented with docstrings. It's very hard to follow along. 
The use of advanced python concepts is nice to see though.